openmdao/core/system.py

```diff
@@ -367,6 +367,10 @@ class System(object):
     _tot_jac : __TotalJacInfo or None
         If a total jacobian is being computed and this is the top level System, this will
         be a reference to the _TotalJacInfo object.
+    _resp_indices_map : dict
```

Comment:
One other feature to try is dynamic coloring. It might need some modifications to support constraint aliases.